web/profiles/custom/yalesites_profile/modules/custom/ys_core/ys_core.module

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains ys_core.module functions.
+ */
+
+/**
+ * Add template file for social links.
+ *
+ * Implements hook_theme().
+ */
+function ys_core_theme($existing, $type, $theme, $path): array {
+  return [
+    'ys_social_links' => [
+      'variables' => [
+        'social_links' => [],
```

Comment:
See above comment. Since this is already a social links component, the wrapper array does not provide value. Instead it adds ambiguity to the shape of the data. https://github.com/yalesites-org/yalesites-project/pull/70/files#r927278582 Suggest changing this to:
```
  return [
    'ys_social_links' => [
      'variables' => [
        'facebook' => NULL,
        'instagram' => NULL,
        ...etc...
      ],
    ],
  ];
```